modules/pool/pool.resolvers.ts

```diff
@@ -49,8 +49,8 @@ const balancerResolvers: Resolvers = {
             }
             return poolService.getPoolJoinExits(args);
         },
-        poolGetFeaturedPoolGroups: async (parent, args, context) => {
-            return poolService.getFeaturedPoolGroups();
+        poolGetFeaturedPoolGroups: async (parent, { chains }, context) => {
+            return poolService.getFeaturedPoolGroups(chains);
```

Comment:
here we would want to do the same that we do e.g. `poolGetAllPoolsSnapshots` below. We have a migration period where we still support both, the header chainId as well as the chains param